tardis/apps/publication_workflow/static/js/lib/angular-material-data-table-0.10.10/dist/md-data-table.js

```diff
@@ -0,0 +1,1486 @@
+        } else {
+          headCtrl.order = scope.getDirection() === 'md-asc' ? scope.orderBy : '-' + scope.orderBy;
+        }
+
+        if(angular.isFunction(headCtrl.onReorder)) {
+          $mdUtil.nextTick(function () {
+            headCtrl.onReorder(headCtrl.order);
+          });
+        }
+      });
+    }
+
+    function updateColumn(index, column) {
+      tableCtrl.$$columns[index] = column;
+
+      if(column.numeric) {
+        element.addClass('md-numeric');
+      } else {
+        element.removeClass('md-numeric');
+      }
+    }
+
+    scope.getDirection = function () {
+      if(isActive()) {
+        return headCtrl.order.charAt(0) === '-' ? 'md-desc' : 'md-asc';
+      }
+
+      return attrs.mdDesc === '' || scope.$eval(attrs.mdDesc) ? 'md-desc' : 'md-asc';
+    };
+
+    scope.$watch(isActive, function (active) {
+      if(active) {
+        element.addClass('md-active');
+      } else {
+        element.removeClass('md-active');
+      }
+    });
+
+    scope.$watch(getIndex, function (index) {
+      updateColumn(index, {'numeric': isNumeric()});
+    });
+
+    scope.$watch(isNumeric, function (numeric) {
+      updateColumn(getIndex(), {'numeric': numeric});
+    });
+
+    scope.$watch('orderBy', function (orderBy) {
+      if(orderBy) {
+        if(!element.hasClass('md-sort')) {
+          enableSorting();
+        }
+      } else if(element.hasClass('md-sort')) {
+        disableSorting();
+      }
+    });
+  }
+
+  return {
+    compile: compile,
+    require: ['^^mdHead', '^^mdTable'],
+    restrict: 'A',
+    scope: {
+      numeric: '=?mdNumeric',
+      orderBy: '@?mdOrderBy'
+    }
+  };
+}
+
+mdColumn.$inject = ['$compile', '$mdUtil'];
+
+angular.module('md.data.table')
+  .decorator('$controller', controllerDecorator)
+  .factory('$mdEditDialog', mdEditDialog);
+
+/*
+ * A decorator for ng.$controller to optionally bind properties to the
+ * controller before invoking the constructor. Stolen from the ngMock.
+ *
+ * https://docs.angularjs.org/api/ngMock/service/$controller
+ */
+function controllerDecorator($delegate) {
+  return function(expression, locals, later, ident) {
+    if(later && typeof later === 'object') {
+      var create = $delegate(expression, locals, true, ident);
+      angular.extend(create.instance, later);
+      return create();
+    }
+    return $delegate(expression, locals, later, ident);
+  };
+}
+
+controllerDecorator.$inject = ['$delegate'];
+  
+function mdEditDialog($compile, $controller, $document, $mdUtil, $q, $rootScope, $templateCache, $templateRequest, $window) {
+  /* jshint validthis: true */
+  
+  var ESCAPE = 27;
+  
+  var busy = false;
+  var body = angular.element($document.prop('body'));
+  
+  /*
+   * bindToController
+   * controller
+   * controllerAs
+   * locals
+   * resolve
+   * scope
+   * targetEvent
+   * template
+   * templateUrl
+   */
+  var defaultOptions = {
+    clickOutsideToClose: true,
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 12 spaces but found 4.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
            clickOutsideToClose: true,
```